src/script/builder/builder.cc

```diff
@@ -80,6 +82,13 @@ ObjectRef DefImpl(String name, ObjectRef obj) {
 
 TVM_REGISTER_NODE_TYPE(BuilderNode);
 
+TVM_REGISTER_GLOBAL("script.builder.Builder").set_body_typed([](){return Builder();});
+TVM_REGISTER_GLOBAL("script.builder.EnterBuilder").set_body_method(&Builder::EnterWithScope);
+TVM_REGISTER_GLOBAL("script.builder.ExitBuilder").set_body_method(&Builder::ExitWithScope);
+TVM_REGISTER_GLOBAL("script.builder.CurrentBuilder").set_body_typed([](){return Builder::Current();});
+TVM_REGISTER_GLOBAL("script.builder.BuilderGet").set_body_method<Builder>(&BuilderNode::Get<ObjectRef>);
```

Comment:
Naming convention: class name + method name

```suggestion
TVM_REGISTER_GLOBAL("script.builder.Builder").set_body_typed([](){return Builder();});
TVM_REGISTER_GLOBAL("script.builder.BuilderEnter").set_body_method(&Builder::EnterWithScope);
TVM_REGISTER_GLOBAL("script.builder.BuilderExit").set_body_method(&Builder::ExitWithScope);
TVM_REGISTER_GLOBAL("script.builder.BuilderCurrent").set_body_typed([](){return Builder::Current();});
TVM_REGISTER_GLOBAL("script.builder.BuilderGet").set_body_method<Builder>(&BuilderNode::Get<ObjectRef>);
```